lib/PuppeteerSharp.Tests/Target/Tests.cs

```diff
@@ -0,0 +1,183 @@
+﻿using Microsoft.AspNetCore.Http;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace PuppeteerSharp.Tests.Target
+{
+    [Collection("PuppeteerLoaderFixture collection")]
+    public class Tests : PuppeteerPageBaseTest
```

Comment:
not sure what to call this class...
in puppeteer it's just
```js
describe('Target', ...
```

Reply:
I'd call this TargetTests